PIL/TiffImagePlugin.py

```diff
@@ -201,6 +201,17 @@
     (II, 2, (1,), 1, (8, 8, 8, 8), (999,)): ("RGBA", "RGBA"),  # Corel Draw 10
     (MM, 2, (1,), 1, (8, 8, 8, 8), (999,)): ("RGBA", "RGBA"),  # Corel Draw 10
 
+    (II, 2, (1,), 1, (16, 16, 16), ()): ("RGB", "RGB;16L"),
```

Comment:
I know, that L means line interleave according to current mode syntax:

> raw mode syntax is "<mode>;<bits><flags>" where "bits" defaults depending on mode (1 for "1", 8 for "P" and "L", etc), and "flags" should be given in alphabetical order.  if both bits and flags have their default values, the ; should be left out
> flags: "I" inverted data; "R" reversed bit order; "B" big endian byte order (default is little endian); "L" line interleave, "S" signed, "F" floating point

But there is already the confusion with "bits": `RGB;16B` means 16-bit per channel big endian, while `RGB;16` means 16 bits per pixel. So, without renaming `RGB;16B` to `RGB;48B`, `RGB;16L` is the best name for this mode, because the mode `16 bits per pixel line interleave` is not possible.

Reply:
I'm ok with this.